CNI/romana

```diff
@@ -184,6 +187,52 @@ set_up_pod () {
 
 	# Kubernetes expects valid json on output so no one above should echo anything in stdout
 	echo -n $R
+
+	update_policy "$POD" "$NAMESPACE"
+}
+
+# update_policy discovers romana network policies that affect current pod
+# and notifies romana agent to include new pod into policy
+update_policy () {
```

Comment:
Wait, maybe we are talking past each other.

Please tell me: Is this function here to cause the application of a policy to a new pod?

If the answer is 'yes' then it's not the policy that's updated, but the pods to which it is applied.

If the answer is 'no' then... well, then please explain.